src/markdown-pages/docs/running-simbank-tests/web-app-integration-test.md

```diff
@@ -0,0 +1,162 @@
+
+![Galasa in hybrid cloud](../../galasa-hybrid-cloud.svg)
+
+The diagram highlights the use of the following technology, and the interaction between the test and those technologies.
+
+ - The 3270 emulator creates a random account number and checks that it does not exist in the CICS region
+ - If the account does not exist the account is created by the Z/OS Batch Manager and is inserted into the backend database
+ - Docker then generates a web page
+ -  Selenium drives the web browser to complete a web form and confirm the balance of the account
+
+
+## Running the WebAppIntegrationTest
+
+As the `WebAppIntetgrationTest` is slightly different to the other tests, there's a bit more set up to do than for the other supplied SimBank tests. 
+
+The test is still run locally but is designed to help you to understand how to set the properties to enable the test to run in a Galasa Ecosystem. When running locally, you can use the localhost setting `127.0.0.1` for increased security. 
+
+The test uses the Selenium Manager, which in turn is dependent on the Docker Manager. Use the following sections to help you to understand how to configure your environment to work with the Selenium and Docker Manager. 
+
+### Using the Selenium Manager
+
+The `WebAppIntetgrationTest` uses the Selenium Manager. To use the Selenium Manager you must have a web browser, for example, Chrome or Firefox installed as well as either a gecko driver or a docker engine. In this example, Firefox and a gecko driver are used. 
+
+You can <a href="https://github.com/mozilla/geckodriver/releases" target="_blank"> download Gecko driver from GitHub</a>. 
+
+You must define the default and local driver properties in the CPS, as shown in the following example:
+
+```
+selenium.default.driver=FIREFOX
+selenium.local.driver.FIREFOX.path=<path/to/geckodriver>
+```
+
+### Using the Docker Manager 
+
+The Selenium Manager has a dependency on the Docker Manager in order to run. Some set up is required for the Docker Manager in the CPS properties file. To configure the Docker Manager, set the following CPS properties:  
+
+```
+docker.dse.engine.PRIMARY=LOCAL
+docker.default.engines=LOCAL
+docker.engine.LOCAL.hostname=127.0.0.1
+docker.engine.local.port=2375
+docker.engine.local.max.slots=10
+docker.container.TAG.name=simbank-webapp
+```
+
+These properties allow local test runs to access the local Docker Engine when the TCP port of the local Docker Engine is enabled.
+
+
+After updating the CPS properties, run the following terminal commands to open a TCP socket for accessing the Dokcer API:
+
+```
+docker pull alpine/socat
+docker run -d -p 127.0.0.1:2376:2375 -v /var/run/docker.sock:/var/run/docker.sock alpine/socat tcp-listen:2375,fork,reuseaddr unix-connect:/var/run/docker.sock
+```
+
+Test that the container is working correctly by running the following command:
+
+```
+mvn install
+docker image build -t simbank-webapp .
+docker run -p 8080:8080 -d simbank-webapp
```

Comment:
this only proves that the docker container is running - they should also view the running web app through a browser